vncdotool/command.py

```diff
@@ -222,7 +223,10 @@ def build_tool(options, args):
     build_command_list(factory, args, options.delay, options.warp)
 
     # connect
-    reactor.connectTCP(options.host, int(options.port), factory)
+    if options.address_family == socket.AF_INET:
```

Comment:
This conditional feels like a code smell.  Perhaps there is a better way to abstract the logic here?

Reply:
Not sure I understand, the `socket.AF_*` constants are pretty standard when it comes to indicating the [Address Family](https://docs.python.org/3/library/socket.html#socket-families) of a socket.

Reply:
Both command and [api.py](https://github.com/sibson/vncdotool/pull/112/files#diff-e45ab1b1ae5f9a01d36a4a4913f9c101R87) have a similar if/then block to create the connection.  Can this be refactored into the client.py class somehow? It seems like it might be a leaky abstraction.  What happens if we add a third type of connection?  It's probably fine for now and not a blocker.

Reply:
Now I got it, good point. I'll fix it asap.